util/design/lib/OtpMemMap.py

```diff
@@ -59,18 +62,35 @@ def _validate_scrambling(scr):
         random_or_hexvalue(dig, "cnst_value", scr["cnst_size"] * 8)
 
 
+def _calc_part_size(part):
+    size = 0
+    print ("working on partition {}".format(part['name']))
```

Comment:
Nit: Can you replace the print statement with `log.info(` here and further below and capitalize the strings to make the messages consistent with all other debug messages in this script?

Reply:
oops sorry. i was debugging something. let me remove this completely.